runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc

```diff
@@ -513,6 +513,24 @@
       "z2": 0
     }
   ],
+  "12597": [ //Juliet's House
```

Comment:
This region is not just Juliet's House. Would be more sensible to call it `Varrock West`

Reply:
Also why is this in the middle of the list?

Reply:
Because it is after "12596" in the list

Reply:
There is no defined order to the file, so putting it in the middle is just really weird

Reply:
I'll change that, thanks for the review